flexget/config_schema.py

```diff
@@ -270,6 +277,13 @@ def is_valid_template(instance):
     return get_template(instance) is not None
 
 
+@format_checker.checks('date', raises=ValueError)
```

Comment:
It doesn't conform to the same standard as dateutil, which means it validates a format that dateutil can't parse...